### DISCUSSION TITLE
请问如何查看产品密钥，买了游戏想要分享，但不知道对应文件路径

### DISCUSSION THREAD

**Discussion by BaeC2004** [pjy612/SteamManifestCache#86] [Question / Answer] [answered]
请问如何查看产品密钥，买了游戏想要分享，但不知道对应文件路径

**Comment by Nobody2613** [comment, marked as answer]
https://github.com/pjy612/SteamManifestCache/wiki/

**Comment by pjy612** [comment]
不是产品密钥（cdkey）
是 游戏（app） 下的 内容仓库（depot） 的 密钥。
[DecryptionKey 密钥](https://github.com/pjy612/SteamManifestCache/wiki/DecryptionKey)
